filesystem/src/jsMain/kotlin/com/xebia/functional/xef/io/NodeCommandExecutor.kt

```diff
@@ -15,6 +15,34 @@ actual val CommandExecutor.Companion.DEFAULT: CommandExecutor
   get() = NodeCommandExecutor
 
 object NodeCommandExecutor : CommandExecutor {
+
+  override suspend fun platform(): Platform {
```

Comment:
I have two questions regarding this:
1. Why is this needed? At the end of the day this always returns `LINUX(something)`, so we could just set this to `LINUX("node")` and call it a day.
2. The implementation seems very brittle. There are modules in Node which do this, and I think delegating on those would be the right move (if getting the platform name is really required).

Reply:
Another possibility is to use the [Platform type](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.native/-platform/) available in the JS target.